### DISCUSSION TITLE
How can I activate copilot as a campus advisor or educational organisation

### DISCUSSION THREAD

**Discussion by iamgoodbytes** [community/community#19410] [Copilot Conversations] [answered]
Hi all, I've been a verified github campus advisor for the school I work at for a couple of years now. For my school I have a verified educational organisation in github.

For the past year I have used and taught my students how they could benefit from co-pilot while coding but I can't find a way to continue to do so. Is there something I need to do to be able to activate co-pilot for free on my account or isn't that possible as a teacher/campus advisor?

Also, I tried reading a similar question on https://github.com/github-community/Global-Campus-Teachers/discussions/92 but when I click the links in my github education feed, I get a 404. Does that mean I lost access or something?

Thanks for helping out!

**Comment by D7EAD** [comment, marked as answer]
Hi! 

According to [Copilot's Documentation](https://docs.github.com/en/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot), "_[i]f you are a student, or a maintainer of a popular open source project on GitHub, you may be eligible for a free GitHub Copilot subscription. If you are not eligible for a free subscription, a paid subscription is available._"

As per Github's official posts regarding the release of Github Copilot, I seem to find no mentions of Copilot being made available to educators nor researchers free-of-charge--at least as of now. I may have missed something, but as a non-student, and considering how recent the release is, I would suggest getting in contact directly with Github Education regarding free-of-charge Copilot availability to institutions, advisors, and educators. You can submit a contact form [here](https://support.github.com/contact/education).

Hopefully that points you in the right direction and, if I've managed to help you, please consider **Marking as Answer** so others can more easily find the solution to this issue if they so encounter it.

**Reply by iamgoodbytes** [reply]
Thank you for the contact form link, I've reached out to github!

**Reply by rwareTHAC** [reply]
Any update?

**Reply by iamgoodbytes** [reply]
@rwareTHAC the reply I got was that teachers or campus advisors don't get free access to co-pilot. For me as a teacher who taught students how to use co-pilot during the past year, that's frustrating because I can't use it myself so I can't show them anymore.

Also, as a teacher I can't use co-pilot but my students can. Should I disallow them from using it in my classes or on exams? It's become a frustrating question now.

**Reply by macbuse** [reply]
I think this sucks personally and I would like to hear what GitHub's justification for this is.
Copilot reduced the time/effort for a lot of boilerplate coding in class. 
So students  can benefit from this but instructors have to pay to check?
In addition Copilot is surely relies for a significant  part of its
knowledge base on repos that are **freely donated** by academics
and their students.

**Reply by laurie** [reply]
👋 

In case you missed the news elsewhere, Copilot is now available for teachers. You can read more here:

https://github.blog/2022-09-08-github-copilot-now-available-for-teachers/

**Reply by jasonralston100** [reply]
You would think they would support teachers. You Know the people responsible for ensuring the future of your company and the Country :)

**Reply by jasonralston100** [reply]
Well, I was wrong, Looks like it is now available free for teachers. :) Good on you 👍
